.gitignore

```diff
@@ -14,5 +14,6 @@ Cargo.lock
 lib*.a
 /docs/_build
 *.iml
+worktree*
```

Comment:
What is this for?

Reply:
I started using `git worktree`s. E.g. a seperate worktree for a clean copy of the main branch such that I can do cherry picking from my messy develop branch and push small PRs.

I prefix the worktree with `worktree` and to avoid that this pops up in the diff, I put it in .gitignore. I hope you don't mind.

Reply:
That should go in your own personal git configuration. The project `.gitignore` is not meant to enable your personal workflow 😄

Reply:
thanks for the hint. I never considered this.